src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -9585,7 +9585,7 @@ void goo()
 }" + TestResources.NetFX.ValueTuple.tuplelib_cs;
 
             // should not crash
-            await VerifyItemExistsAsync(markup, "ToString");
+            await VerifyNoItemsExistAsync(markup);
```

Comment:
this test was bogus.  we shoudl not have offered completion here.  teh cleanup simplified code and ended up fixing this.